Project.toml

```diff
@@ -1,17 +1,10 @@
 name = "GoogleDrive"
 uuid = "91feb7a0-3508-11ea-1e8e-afea2c1c9a19"
 authors = ["tejasvaidhyadev <iamtejasvaidhya@gmail.com> and contributors"]
-version = "0.1.3"
+version = "0.2.0"
 
 [deps]
-DataDeps = "124859b0-ceae-595e-8997-d05f6a7a8dfe"
-Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
 Downloads = "f43a241f-c20a-4ad4-852c-f6b1247861c6"
-HTTP = "cd3eb016-35fb-5094-929b-558a96fad6f3"
-Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 
 [compat]
-DataDeps = "0.7.10"
-Downloads = "1"
```

Comment:
I think we should retain
`Downloads = "1"`
because of the new guideline:
https://discourse.julialang.org/t/psa-compat-requirements-in-the-general-registry-are-changing/104958